pkg/package.go

```diff
@@ -235,3 +235,14 @@ func (p Package) DownloadWithChanged(localFile string) (bool, error) {
 	}
 	return false, nil
 }
+
+func (p Package) Copy(destInstance *Instance) error {
```

Comment:
where is `CopyWithChanged` ? :)

if source package is already on target instance then copying could be skipped ;)

Reply:
I am dreaming for checksums calculated and served by crx/packmgr then we could leverage that info and implement deploy skipping without local file locks